### PR TITLE
export: fixes export problem

### PR DESCRIPTION
**Pull request by ioannistsanaktsidis**

### CODE REVIEW

**Comment by jmartinm** [comment]
With the indentation changes it is difficult to see what are the 'real' changes.

I see you introduced `MINIMUM_PAGE_LIMIT ` what is the use of it?

File: inspire/base/static/js/search/search_results.js [hunk 1]
```diff
@@ -120,14 +121,20 @@ define(
             sList.push($(this).attr("id"));
           });
 
-          if (parseInt($('#total-results').text()) > EXPORT_LIMIT) {
-            $('#results-panel').after('<div class="panel panel-default" id="info-message">' +
-              '<div class="panel-body" >You have selected ' + sList.length + ' records of this page. <a class="pointer" id="select-all-records">' +
-              'Select ' + EXPORT_LIMIT + ' records (Maximum limit).</a></div></div>');
+
+          if (parseInt($('#total-results').text()) > this.attr.MINIMUM_PAGE_LIMIT) {
```
Comment (jmartinm) [inline inspire/base/static/js/search/search_results.js]: This value will be taken from the pagination dropdown to get the correct value.

**Comment by ioannistsanaktsidis** [comment]
PR updated so as to take the value from the dropdown for results per page.

**Comment by ioannistsanaktsidis** [comment]
@jmartinm the PR is amended and ready for merging. Cheers.

**Comment by jmartinm** [comment]
Merged c28b48a5c7cc12f1f36204dd7f1fe023e3f7ec11

**Pull request closed without merging**